<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btOccupancy</class>
 <widget class="QMainWindow" name="btOccupancy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>748</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>748</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>748</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Occupancy Tool For Multi Sensors</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="imageView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>641</width>
      <height>721</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="noImageLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>30</red>
          <green>30</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>20</red>
          <green>19</green>
          <blue>18</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>No Image Loaded</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="btListWidget" name="listData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>101</width>
      <height>501</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkAddSelect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adding</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionRead_TRK_File"/>
    <addaction name="actionRead_GRD_File"/>
    <addaction name="actionRead_GRD_Dir"/>
    <addaction name="actionRead_OCC_File"/>
    <addaction name="actionExport_OCC_File"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>&amp;Map</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionClearImage"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionExport_View"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>&amp;Configuration</string>
    </property>
    <addaction name="actionLoadParameters"/>
    <addaction name="actionLoadDevices"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuConfig"/>
   <addaction name="menuMap"/>
  </widget>
  <action name="actionRead_HST_File">
   <property name="text">
    <string>&amp;Read HST File</string>
   </property>
  </action>
  <action name="actionLoadImage">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionClearImage">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionDelete_Data">
   <property name="text">
    <string>&amp;Delete Data</string>
   </property>
  </action>
  <action name="actionRead_GRD_File">
   <property name="text">
    <string>Read &amp;GRD File</string>
   </property>
  </action>
  <action name="actionLoadParameters">
   <property name="text">
    <string>Load Parameters</string>
   </property>
  </action>
  <action name="actionLoadDevices">
   <property name="text">
    <string>Load Devices</string>
   </property>
  </action>
  <action name="actionRead_TRK_File">
   <property name="text">
    <string>Read TRK File</string>
   </property>
  </action>
  <action name="actionExport_OCC_File">
   <property name="text">
    <string>Export OCC File</string>
   </property>
  </action>
  <action name="actionRead_OCC_File">
   <property name="text">
    <string>Read OCC File</string>
   </property>
  </action>
  <action name="actionRead_GRD_Dir">
   <property name="text">
    <string>Read GRD Dir</string>
   </property>
  </action>
  <action name="actionExport_View">
   <property name="text">
    <string>Export View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>btListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">btlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
